advanced_filters/admin.py

```diff
@@ -56,15 +82,24 @@ def __init__(self, *args, **kwargs):
             self.original_change_list_template = "admin/change_list.html"
         self.change_list_template = self.advanced_change_list_template
         # add list filters to filters
-        self.list_filter = (AdvancedListFilters,) + tuple(self.list_filter)
-
-    def save_advanced_filter(self, request, form):
+        self.list_filter = (AdvancedListFilters, AdvancedQueryFilters,) + tuple(self.list_filter)
+
+    def save_advanced_filter(self, request, form, save_instance):
+        if '_just_filter' in request.POST:
+            search_query = form.generate_query()
+            query_serializer = QSerializer(base64=True)
+            b64_query = query_serializer.dumps(search_query)
+            url = "{path}?_aquery={query}"\
+                .format(path=request.path, query=b64_query)
+            return HttpResponseRedirect(url)
```

Comment:
How about we add this into it's own method  `filter_without_saving` or `filter_ad_hoc` or some other name, and allow it to be called from `adv_filters_handle` if not saving.